packaging/make_deb.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
+
+DISTRIBUTION="_$1"
+echo $DISTRIBUTION
+
+# Compress the changelog, strip the binaries
+cp changelog.Debian calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/changelog.Debian
+# Options : -f for removing file if it was there, -n for no time stamp
+gzip -9 calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/changelog.Debian -f -n
```

Comment:
I did not know what `-9` is for (best compression). Maybe use `--best`, instead, as it is a bit more intuitive?
```suggestion
# Options : --best for best compression, -f for removing file if it was there, -n for no time stamp
gzip --best -f -n calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/changelog.Debian
```

(same below)